c++/include/orc/Writer.hh

```diff
@@ -38,6 +38,11 @@ namespace orc {
     CompressionStrategy_COMPRESSION
   };
 
+  enum RleVersion {
+    RleVersion_1,
```

Comment:
You may want to explicitly assign values here to ensure the addition of new encodings or refactoring work can't unintentionally change the underlying integer values.